fcl/reco/Stage1/Run2/stage1_run2_icarus_MC_slimout.fcl

```diff
@@ -0,0 +1,118 @@
+            mcopflashTPC3:                  @local::ICARUSMCOpFlashTPC3
+          
+            cheatopflashTPC0:               @local::ICARUSCheatOpFlashTPC0
+            cheatopflashTPC1:               @local::ICARUSCheatOpFlashTPC1
+            cheatopflashTPC2:               @local::ICARUSCheatOpFlashTPC2
+            cheatopflashTPC3:               @local::ICARUSCheatOpFlashTPC3
+
+            ### mc producers
+            mchitfinder:                    @local::standard_mchitfinder
+            mcassociationsGausCryoE:        @local::standard_mcparticlehitmatching
+            mcassociationsGausCryoW:        @local::standard_mcparticlehitmatching
+}
+
+physics.reco: [ 
+                @sequence::icarus_reco_Gauss_CryoE ,
+                @sequence::icarus_reco_Gauss_CryoW ,
+		@sequence::icarus_reco_fm,
+                caloskimCalorimetryCryoE, caloskimCalorimetryCryoW,
+                mcassociationsGausCryoE,  mcassociationsGausCryoW
+              ]
+
+# Turn on truth-info for track skimmer 
+physics.analyzers.caloskimE.G4producer: "largeant"
+physics.analyzers.caloskimE.SimChannelproducer: "largeant"
+physics.analyzers.caloskimE.RawDigitproducers: ["MCDecodeTPCROI:PHYSCRATEDATATPCEW", "MCDecodeTPCROI:PHYSCRATEDATATPCEE"]
+physics.analyzers.caloskimW.G4producer: "largeant"
+physics.analyzers.caloskimW.SimChannelproducer: "largeant"
+physics.analyzers.caloskimW.RawDigitproducers: ["MCDecodeTPCROI:PHYSCRATEDATATPCWW", "MCDecodeTPCROI:PHYSCRATEDATATPCWE"]
+
+physics.outana:            [ caloskimE, caloskimW, simpleLightAna, CRTDataAnalysis]
+physics.trigger_paths:     [ reco ]
+physics.end_paths:         [ outana, stream1 ]
+outputs.out1.fileName:     "%ifb_%tc-%p.root"
+outputs.out1.dataTier:     "reconstructed"
+outputs.out1.SelectEvents: [ reco ]
+
+outputs.out1.outputCommands: [
+  "keep *_*_*_*",
+  "drop *_caloskimCalorimetryCryoE_*_*",
+  "drop *_caloskimCalorimetryCryoW_*_*",
+  # Drop more stuff to reduce filesize output
+  "drop recob::Wire*_decon2droi*_*_*",
+  "drop recob::Wire*_roifinder2d_*_*",
+  # "drop recob::Wire*_roifinder_*_*",
+  "drop sim::SimPhotons*_*_*_*",
+  "drop sim::SimEnergyDeposit*_*_*_*",
+  "drop sim::SimChannel*_largeant_*_*",
+  "drop raw::OpDetWaveform*_*_*_*",
+  "drop recob::OpHit*_mcophit_*_*"
+]
+
+# Disabled Space-Charge service for calorimetry
+services.SpaceChargeService: {
+    EnableCalEfieldSCE: false
+    EnableCalSpatialSCE: false
+    EnableCorrSCE: false
+    EnableSimEfieldSCE: false
+    EnableSimSpatialSCE: false
+    InputFilename: "SCEoffsets/SCEoffsets_ICARUS_E500_voxelTH3.root"
+    RepresentationType: "Voxelized_TH3"
+    service_provider: "SpaceChargeServiceICARUS"
+}
+
+services.BackTrackerService:        @local::standard_backtrackerservice
+services.ParticleInventoryService:  @local::standard_particleinventoryservice
+
+# Use this to take out the interplane offsets that are put in during the G4 stage
+physics.producers.cluster3DCryoE.Hit3DBuilderAlg.UseT0Offsets: true
```

Comment:
Was this fhicl based off the `stage1_run2_icarus_MC.fcl`?

Do we want to use this and not the "WC" one? (Maybe it's related to using 1d vs 2d deconvolution outputs downstream, but anyway curious the provenance.)

Related to the answers above, @SFBayLaser should comment, but based on the comment alone here I'm wondering if we want these to be `false` since the simulation removes time offsets?

Reply:
The "wc" configs use the output of the 2D deconvolution, which is not the nominal configuration an is not what is needed here.

Reply:
Okay then I think my question for @SFBayLaser about this setting is valid. Tracy, do we want this to be false for running where we expect the interplane differences to be removed upstream or is it okay the way it is?

Reply:
This is now addressed by https://github.com/SBNSoftware/icaruscode/pull/542.